libnavui-maneuver/src/main/res/drawable/mapbox_exit_board_background.xml

```diff
@@ -1,16 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle">
-  <stroke
-      android:width="1dp"
-      android:color="@android:color/black" />
-
-  <solid android:color="@android:color/white" />
-
-  <padding
-      android:left="4dp"
-      android:right="4dp"
-      android:bottom="4dp"
-      android:top="4dp" />
-
-  <corners android:radius="5dp" />
-</shape>
\ No newline at end of file
+    <stroke android:width="1dp" android:color="@android:color/black"/>
+    <solid android:color="@android:color/white"/>
+    <padding android:left="4dp" android:right="4dp" android:bottom="4dp" android:top="4dp"/>
+    <corners android:radius="5dp"/>
+</shape>
```

Comment:
This seems unrelated. Why do we need these changes?
They contradict Android Studio formatter (Code -> Reformat Code).

Reply:
See [the comment](https://github.com/mapbox/mapbox-navigation-android/pull/5312/files#r774981365)

Reply:
fixed